py3status/modules/battery_level.py

```diff
@@ -237,8 +251,69 @@ def _refresh_battery_info(self):
         acpi_list = [acpi_list[i:i + 2]
                      for i in range(0, len(acpi_list) - 1, 2)]
 
-        battery_list = [self._extract_battery_information_from_acpi(battery)
-                        for battery in acpi_list]
+        return [_parse_battery_info(battery) for battery in acpi_list]
+
+    def _extract_battery_information_from_sys(self):
+        """
+        Extract the percent charged, charging state, time remaining,
+        and capacity for a battery, using Linux's kernel /sys interface
+
+        Only available in kernel 2.6.24(?) and newer. Before kernel provided
+        a similar, yet incompatible interface in /proc
+        """
+        def _parse_battery_info(sys_path):
+            """
+            Extract battery information from uevent file, already convert to
+            int if necessary
+            """
+            raw_values = dict()
+            with open(sys_path + "/uevent") as f:
+                for var in f.read().split():
```

Comment:
use `splitlines()` not `split()` here as my battery has a space in the serial number which causes an error.